### DISCUSSION TITLE
what is the default value of "perf.type" in function rfsrc?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#170] [Q&A]
**what is the default value of "perf.type" in function rfsrc?  The help doesn't seem to say that**
![image](https://user-images.githubusercontent.com/5112397/149283532-e09ab4f1-711b-459e-bc4c-c92f9dbfb69d.png)

**Comment by ishwaran** [comment]
The default is `perf.type="misclass"`.  I will add this information to the help file.  Thanks.
